src/helpers/isHotPost.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  minimumLikes,
+  minimumNumberOfComments,
+  minimumParticipants,
+  minimumViews,
+  relevancePeriod,
+} from '@/data/hotPost'
+import axios from 'axios'
+import env from '@/helpers/env'
+import getFeedsContract from '@/helpers/getFeedsContract'
+
+async function getParticipants(feedId: number, postId: number) {
+  const [{ participants }] = await getFeedsContract.getPostsAndParticipants(
+    feedId,
+    postId,
+    1,
+    false
```

Comment:
What is `1` and `false` for? We should probably make a wrapper with named arguments to make the code more readable